dom.bs

```diff
@@ -209,27 +209,62 @@ against a <var>set</var>, run these steps:
 added.
 
 
-<h3 id=namespaces>Namespaces</h3>
+<h3 id=name-validation oldids=namespaces>Name validation</h3>
 
-<p>To <dfn export>validate</dfn> a <var>qualifiedName</var>, <a>throw</a> an
-"{{InvalidCharacterError!!exception}}" {{DOMException}} if <var>qualifiedName</var> does not match
-the <code><a type>QName</a></code> production.
+A [=string=] |string| is a <dfn>valid basic name</dfn> if it does not contain
+[=ASCII whitespace=], U+0000 NULL, U+002F (/), or U+003E (>).
 
-<p>To <dfn export>validate and extract</dfn> a <var>namespace</var> and <var>qualifiedName</var>,
-run these steps:
+<div class="note">
+ <p>This concept is used to validate [=attribute=] [=Attr/local names=] and
+ [=Attr/qualified names=], as well as both [=Element/namespace prefix|element=] and
+ [=Attr/namespace prefix|attribute=] namespace prefixes, in the cases where elements and attributes
+ are constructed with DOM APIs. Note that the prefixes additionally cannot contain U+003A (:) by
+ construction since they always result from the [=validate and extract=] algorithm.
+
+ <p>Note that this means [=attribute=] [=Attr/local names=] for attributes constructed with DOM APIs
+ can contain U+003D (=) in positions beyond the initial character, which is not possible when using
+ the HTML parser.
```

Comment:
If we didn't think there would be a compat issue, it seems simpler to just outlaw `=` entirely here, rather than have this very special behavior. But also ok as-is.